### PR TITLE
Biocollect 94

### PR DESCRIPTION
**Pull request by sat01a**

### CODE REVIEW

File: grails-app/controllers/au/org/ala/biocollect/ProjectActivityController.groovy [hunk 1]
```diff
@@ -63,6 +65,40 @@ class ProjectActivityController {
...
     }
 
+    /**
+     * Delete activities and records by project activityId
+     * @param id projact activity id
+     * @return
+     */
+
+    def unpublish(String id) {
+        def pActivity = projectActivityService.get(params.id)
```
Comment (m-r-c) [inline grails-app/controllers/au/org/ala/biocollect/ProjectActivityController.groovy]: Use types where known (String)

File: grails-app/views/projectActivity/_info.gsp [hunk 2]
```diff
@@ -21,11 +28,16 @@
             </div>
 
             <div class="row-fluid">
-                <div class="span3 text-right">
-                    <label class="control-label" for="description">Description: <span class="req-field"></span></label>
+                <div class="span4 text-right">
+                    <label class="control-label" for="description">Description:
```
Comment (m-r-c) [inline grails-app/views/projectActivity/_info.gsp]: Use g:message for label text

File: grails-app/views/projectActivity/_status.gsp [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<label>
```
Comment (m-r-c) [inline grails-app/views/projectActivity/_status.gsp]: This should be a div or span, not a label: labels are for labelling input fields.

**Comment by m-r-c** [comment]
Can we move all the Save, Next, Prev buttons to below the grey boxes so they are consistent? It would probably be nicer if the buttons on the Species tab were inside the box, but that would be difficult given the behaviour of that tab.

File: web-app/js/projectActivityInfo.js [hunk 4]
```diff
@@ -78,4 +78,14 @@ var pActivityInfo = function(o, selected){
...
     }
+
+    self.isInfoValid = function () {
+        return self.name() && self.description() && self.startDate() && self.isEndDateAfterStartDate();
+    };
+
+    self.isEndDateAfterStartDate = function () {
+        var start = moment(self.startDate());
+        var end = moment(self.endDate());
+        return end >= start;
```
Comment (m-r-c) [inline web-app/js/projectActivityInfo.js]: This check doesn't work if the end date is not set: it always results in the error message being displayed.

File: grails-app/views/projectActivity/_info.gsp (hunk 2 above)
Comment (sat01a) [inline grails-app/views/projectActivity/_info.gsp]: fixed

File: grails-app/views/projectActivity/_status.gsp (hunk 3 above)
Comment (sat01a) [inline grails-app/views/projectActivity/_status.gsp]: used span instead of label

File: web-app/js/projectActivityInfo.js (hunk 4 above)
Comment (sat01a) [inline web-app/js/projectActivityInfo.js]: Good one, fixed it. 
Also moved Save,Next and Prev out of grey area.

**Pull request merged**